recipe/meta.yaml

```diff
@@ -18,38 +21,54 @@ outputs:
     build:
       noarch: python
     requirements:
-      build:
+      host:
         - python >=3.6
         - pip
       run:
         - python >=3.6
-        - numpy >=1.9.1
-        - scipy >=0.14
-        - pyyaml
-        - h5py
-        - tensorflow >=2.2
+        # - numpy >=1.9.1
+        # - scipy >=0.14
+        # - pyyaml
+        # - h5py
+        # - tensorflow >=2.2
       # add run_constrained to prevent installation of mismatched keras and
       # keras-base packages
       run_constrained:
-        - keras {{ version }}
+        # - keras {{ version }}
+        - tensorflow {{ version }}
     test:
-      imports:
-        - keras
-        - keras.backend
-        - keras.datasets
-        - keras.engine
-        - keras.layers
-        - keras.preprocessing
-        - keras.utils
-        - keras.wrappers
+      requires:
+        - pip
+      commands:
+        - pip check
+      # imports:
+        # - keras
+        # - keras.backend
+        # - keras.datasets
+        # - keras.engine
+        # - keras.layers
+        # - keras.preprocessing
+        # - keras.utils
+        # - keras.wrappers
 
   - name: keras
+    script: install_base.sh  # [not win]
+    script: install_base.bat # [win]
     build:
       noarch: true
     requirements:
+      host:
+        - python >=3.6
+        - pip
       run:
-        - keras-base {{ version }}
-        - tensorflow
+        - python >=3.6
+      run_constrained:
```

Comment:
see above

Reply:
The constrains for tensorflow were removed, only the constraint for keras was left on the recipe
https://github.com/AnacondaRecipes/keras-feedstock/blob/update_to_2.6.0/recipe/meta.yaml#L66&L68